### PR TITLE
docs: added experience share from Apache APISIX committer blog 

### PR DESCRIPTION
**Pull request by Serendipity96**
docs: added experience share from Apache APISIX committer blog

### CODE REVIEW

File: _posts/blog/zh-CN/experience-share-from-Apache-APISIX-committer.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+
+本次采访邀请到了去年参加编程之夏的两位同学，两位同学现在都是 Apache APISIX 的 committer，也是今年社区项目的导师，分享正式开始！
+
+### 分享嘉宾：吴舒旸
+
+![yiyiyimu](../static/images/2021-06-03-1.webp)
+
+**Q：舒旸，请简单介绍一下自己。**
+
+**舒旸**：Hello，大家好~ 我是吴舒旸（Github ID: yiyiyimu），去年本科毕业于上海交通大学环境学院，今年八月会在佐治亚理工学院开始读计算机硕士。现在在 APISIX 背后的商业公司——支流科技全职维护开源社区，也是 Apache APISIX 和 Chaos Mesh 两个项目的 committer。去年参与“编程之夏 2020”在 Apache APISIX 完成了“支持 etcd v3 协议”这个项目，这也作为了 APISIX v2.0 的主要更新点，之后也有幸被评上了“编程之夏 2020”的优秀学生，今年也会作为 mentor 继续参与“编程之夏 2021”。
```
Comment (yzeng25) [inline _posts/blog/zh-CN/experience-share-from-Apache-APISIX-committer.md:19]: APISIX --- > Apache APISIX

File: _posts/blog/zh-CN/experience-share-from-Apache-APISIX-committer.md [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+
+**舒旸**：我把去年会议上讲的 PPT 截出来了，对于有明确兴趣目标的同学，自然不需要考虑如何选择项目的问题。但如果像去年的我自己一样，看一圈项目下来发现都不怎么看得懂该怎么办呢。这个时候大家可能会去选择 star 数多的项目，但实际上 **star 数多只表明了这个项目的受欢迎程度，而不是能从社区中获得多少帮助的“社区活跃度”指标。活跃的社区在你两个阶段都会有所裨益，在入门时提供足够的帮助，也在后面维护社区时提供足够的反馈。**
+
+![github introduction](../static/images/2021-06-03-2.webp)
+
+**舒旸**：**我当时选择的指标主要是 issue 回复情况（入门时碰到问题能否及时得到回复）、good first issue 数量（有没有好入手的 issue，既增加对项目的了解也可以多刷脸）以及 insights 指标（社区整体的活跃度）。** 今年我们在公司也写了一个查看[贡献者](https://github.com/api7/contributor-graph)情况的小工具，可以更直观的看到社区贡献者增长情况以及月度活跃贡献者多少，也是一个不错的选择指标。
+
+![join opensource](../static/images/2021-06-03-3.webp)
+
+**舒旸**：去年参与 APISIX 之前我对于项目需要的技术栈包括 Lua/Openresty/etcd 甚至 API 网关都完全没有任何了解，但其实上手速度很快。借助文档把程序跑起来运行几个 demo 很快就可以完成，Lua 也半天就可以上手。这里再推荐一下 APISIX PMC moonming 大佬的两份 Openresty 入门必备，免费的 Openresty 电子书和付费的 Openresty 极客时间课程，当时入门从其中收获颇丰。
```
Comment (yzeng25) [inline _posts/blog/zh-CN/experience-share-from-Apache-APISIX-committer.md:35]: APISIX --- > Apache APISIX

Comment (yzeng25) [inline _posts/blog/zh-CN/experience-share-from-Apache-APISIX-committer.md:35]: consider add a hyperlink to github page here?

**Review by yzeng25 — CHANGES_REQUESTED** [review]

File: _posts/blog/zh-CN/experience-share-from-Apache-APISIX-committer.md (hunk 1 above)
Comment (Serendipity96) [reply _posts/blog/zh-CN/experience-share-from-Apache-APISIX-committer.md:19]: thanks

**Pull request closed without merging**